packages/components/src/components/hds/table/index.hbs

```diff
@@ -13,6 +13,9 @@
   <thead class="hds-table__thead">
     {{#if @columns}}
       <Hds::Table::Tr
+        @sortOrder={{if (eq "__selected__" this.sortBy) this.sortOrder}}
```

Comment:
Will probably end up just comparing this to the selectedItemKey directly instead of using this string constant.